rodan/jobs/core.py

```diff
@@ -1003,9 +1004,17 @@ def create_archive(resource_uuids):
                 print("{} has no file!".format(resource.name))
                 continue
 
+            # determine a path that doesn't conflict
+            filepath = resource.name + "." + resource.resource_type.extension
+            if filepath in archive.namelist():
+                for i in xrange(1, sys.maxint):
```

Comment:
Using an iterable object is good, but `xrange` became `range` in `python3`. To not have things break later and have the same code for python2/python3 that we must do right now, do one of the following please.

- Either do this to overwrite the old `range` in python2 .

```python
from builtins import range
for i in range...
```
- Or import `xrange` in python3
```python
from past.builtins import xrange
for i in xrange...
```

Reply:
```python
for i in six.moves.range(1, sys.maxint): 
  ...
```